chan-plugin/chan-plugin.php

```diff
@@ -0,0 +1,50 @@
+<?php /**
+ * Plugin Name: chan-plugin
+ * Plugin URI: ''
+ * Author: Chandan
+ * Author URI: ''
+ * Version: 2.3.5
+ * Description: 'A new Plugin'
+ * Text Domain: 'xyz'
+ * *
+ * 
+ */
+
+ //Function to handle a shortcode
+
+function handle_shortcode( $attr, $content='' ){
```

Comment:
use document block for function documentation

Example https://d.pr/i/q9uOp7

Reply:
Document blocks has been added